+srv/expServer.m

```diff
@@ -358,19 +359,30 @@ function whiteScreen()
 
   function calibrateGamma()
     stimWindow = rig.stimWindow;
+    % move PTB window to background and bring MATLAB command window to front
+    Screen('Preference', 'WindowShieldingLevel', 0);
+    stimWindow.close();
```

Comment:
Why close and open stimWindow?

Reply:
currently open PTB windows are not affected when `'WindowShieldingLevel'` changes